jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter.bats

```diff
@@ -51,3 +57,52 @@ teardown() {
     unset JAVA_INITIAL_MEM_RATIO
     unset JAVA_MAX_MEM_RATIO
 }
+
+@test "verify the env vars when LOG_LEVEL is set" {
+    LOG_LEVEL="WARN"
+    LOGGER_CATEGORIES=org.xyz=INFO,com.acme=DEBUG
+    configure_logger_config_file
+    [[ "${LOG_LEVEL}" == WARN ]]
```

Comment:
this test does not do too much, can you please improve it?
LOG_LEVEL are being set in this tests and being tested with the same value.

Reply:
Done